scripts/run_e2e_tests.py

```diff
@@ -128,6 +128,33 @@ def is_oppia_server_already_running() -> bool:
     return False
 
 
+def run_ng_compilation() -> None:
+    """Runs webpack compilation.
```

Comment:
Shouldn't be it angular CLI compilation.?

Reply:
Done!